app/src/main/java/Fragments/ProfileFragment.java

```diff
@@ -22,17 +48,105 @@ public static ProfileFragment newInstance() {
     }
 
     @Override
-    public void onCreate(Bundle savedInstanceState)
-    {
+    public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
     }
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_profile, container, false);
+
+        name_tv = (TextView) view.findViewById(R.id.profile_name);
+        email_tv = (TextView) view.findViewById(R.id.profile_email);
+        phoneNumber_tv = (TextView) view.findViewById(R.id.profile_phone_number);
+        photo_iv = (ImageView) view.findViewById(R.id.profile_photo);
+        addPhoto_ib = (ImageButton) view.findViewById(R.id.profile_add_photo);
+
```

Comment:
Just wondering: what does _tv, _iv and _ib stand for ?

Reply:
textView, imageView and imageButton